simplivity/resources/policies.py

```diff
@@ -89,6 +92,26 @@ def create(self, name, flags=None, timeout=-1):
         affected_object = self._client.do_post(URL, data, timeout, flags=flags)[0]
         return self.get_by_id(affected_object["object_id"])
 
+    def suspend(self, target=None, timeout=-1):
+        """Suspends policy-based backups on a specific targeted object
+            Args:
+                target: Target object.
+                        Allowed object of host, omnistack_cluster, cluster_group and default is federation
+                timeout: Time out for the request in seconds.
+
+            Returns:
+                None
+        """
+        data = {}
+        if isinstance(target, (Host, OmnistackCluster, ClusterGroup)):
+            data["target_object_type"] = target.OBJECT_TYPE
```

Comment:
As we can easily find the object type using isinstance() or type(),  I don't think its a good idea to have an extra variable just to check the object type.

Reply:
@sijeesh , We tried using type function to get the object type however value is simplivity.resources.omnistack_clusters.OmnistackCluster, whereas we need the value as 'omnistack_cluster' so we cannot really use type function also isinstance gives us boolean output. In summary we were not able to use the programmatic functions to retrieve the value and hence we declared OBJECT_TYPE variable.

Reply:
There needs to be a way to get the DATA_FIELD for the response type of the object for the refresh method(s).  In this case, it was called OBJECT_TYPE.

 Should we encourage the use of an ENUM instead?

Reply:
@svt-hchaudhari, Here we can use a map/enum/if condition to get the object type.

@svt-kmcdaniel, It makes sense if the OBJECT_TYPE class variable is useful in the resource refresh method to get the data from the GET response body. I was thinking this variable is added only for policy's suspend and resume methods.
@svt-hchaudhari, in this case, let's use this variable here.